### PR TITLE
feat: added option to customize job parameters

### PR DESCRIPTION
**Pull request by CptSchnitz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,4 +30,9 @@ The queue name is: `tiles-${projectName}`
 - APP_ENABLE_REQUEST_QUEUE_HANDLING - should this instance work on the request queue, or only api
 - APP_REQUEST_QUEUE_CHECK_INTERVAL_SEC
 - APP_TILES_BATCH_SIZE - how many tiles should be pushed into the queue at once
-- APP_METATILE_SIZE
\ No newline at end of file
+- APP_METATILE_SIZE
+- DB_JOBS_EXPIRE_IN_SECONDS - How many seconds a job may be in active state before it is failed because of expiration. Must be >=1
+- DB_JOBS_RETRY_BACKOFF - Default: true. Enables exponential backoff retries based on retryDelay instead of a fixed delay. Sets initial retryDelay to 1 if not set.
+- DB_JOBS_RETRY_LIMIT - Max number of retries of failed jobs. Default is 3 retries.
+- DB_JOBS_RETRY_DELAY - Default: 60. Delay between retries of failed jobs, in seconds.
```
Comment (melancholiai) [inline README.md]: I know `pgboss` called this option `retryDelay` but let's be consistent with adding the time unit

File: config/custom-environment-variables.json [hunk 2]
```diff
@@ -76,6 +76,26 @@
...
+      "__format": "number"
+    },
+    "retryDelay": {
+      "__name": "DB_JOBS_RETRY_DELAY",
+      "__format": "number"
+    },
+    "retentionHours": {
+      "__name": "DB_JOBS_RETENTION_HOURS",
+      "__format": "number"
+    }
```
Comment (melancholiai) [inline config/custom-environment-variables.json]: nest under jobs

**Review by melancholiai — CHANGES_REQUESTED** [review]

File: config/custom-environment-variables.json (hunk 2 above)
Comment (CptSchnitz) [reply config/custom-environment-variables.json]: no, it's more work

File: README.md (hunk 1 above)
Comment (melancholiai) [inline README.md]: specify the default value of each variable

**Review by melancholiai — APPROVED** [review]

**Pull request merged**